node/dispatch.go

```diff
@@ -0,0 +1,70 @@
+package node
+
+import (
+	"fmt"
+	"sync"
+)
+
+// This file holds a global structure used for dispatching
+// NOTE: currently this is developed for PC MVP Shift and may be phased out depending on performance and need
+
+var (
+	dispatchPeers map[Blockchain]map[string]Node	// <BlockchainOBJ><GID><Node>
+	m sync.Mutex
+	one sync.Once
+)
+
+func getDispatchPeers() map[Blockchain]map[string]Node{
+	one.Do(func() {
+		dispatchPeers = make(map[Blockchain]map[string]Node)
+	})
+	return dispatchPeers
+}
+
+
+func NewDispatchPeer(newNode Node){
+	m.Lock()
+	defer m.Unlock()
+	dispatchPeers := getDispatchPeers()
+	for _,blockchain := range newNode.Blockchains{
+		nodes := dispatchPeers[blockchain] // type map[GID]Node
+		if nodes == nil { // blockchain not within list
+			dispatchPeers[blockchain] = map[string]Node{newNode.GID:newNode}// add new node to empty map
+		}else{	// blockchain is within list
+			nodes[newNode.GID]=newNode // add node to inner map
```

Comment:
Formatting of this line is incorrect should be:
`nodes[newNode.GID] = newNode`

Reply:
Yes, as discussed with irving, I'm going to reformat everything from tabs to spaces and change all of my comments to fit the official go spec. See #106

Reply:
That way we can easily run gofmt or goimports and it won't mess up our comments